docs/src/service-worker-network-interception.md

```diff
@@ -0,0 +1,101 @@
+---
+id: service-worker-network-interception
+title: "Service Worker Network Interception (Chromium)"
+---
+
+:::note
+Service Worker Network instrumentation only works in Chrome / Chromium. If you want full network interception in Firefox and WebKit, you can disable Service Workers by setting [`option: Browser.newContext.serviceWorkers`] to `'block'`.
+:::
+
+If a site uses [Service Workers](https://developer.mozilla.org/en-US/docs/Web/API/Service_Worker_API), Playwright's standard set of Network Events and Interception/Routing will work. Service Workers have complex lifecycles and can intercept Network Requests themselves, so there are a few things to note.
+
+## Event Targets
```

Comment:
This section explains how service worker fetch works. Let's name it accordingly?